src/core/comm/src/4C_comm_pack_helpers.hpp

```diff
@@ -86,6 +86,24 @@ namespace Core::Communication
    */
   void add_to_pack(PackBuffer& data, const Packable auto& obj) { obj.pack(data); }
 
+  /*!
+   * \brief Add stuff to the end of a char vector data
+   *
+   * This method is templated for std::optional<T>
+   * \param[in,out] data char string stuff shall be added to
+   * \param[in] stuff std::optional<T> that get's added to stuff
+   */
```

Comment:
```suggestion
  /*!
   * \brief Add stuff to the end of a char vector data
   *
   * This method is templated for std::optional<T>
   * \param[in,out] data char string stuff shall be added to
   * \param[in] stuff std::optional<T> that gets added to stuff
   */
```

Given that you fixed doxygen: it would probably make the most sense to document these functions once and refer to a general doc for all the overloads. Just a thought; I might do that.